src/main/kotlin/no/nav/syfo/client/oppfolgingstilfelle/OppfolgingstilfellePersonDTO.kt

```diff
@@ -0,0 +1,33 @@
+package no.nav.syfo.client.oppfolgingstilfelle
+
+import java.time.LocalDate
+
+data class OppfolgingstilfellePersonDTO(
+    val oppfolgingstilfelleList: List<OppfolgingstilfelleDTO>,
+    val personIdent: String,
+)
+
+data class OppfolgingstilfelleDTO(
+    val arbeidstakerAtTilfelleEnd: Boolean,
+    val start: LocalDate,
+    val end: LocalDate,
+    val virksomhetsnummerList: List<String>,
+)
+
+fun OppfolgingstilfellePersonDTO.toLatestOppfolgingstilfelle(): Oppfolgingstilfelle? {
```

Comment:
Kan vurdere å lage en egen enhetstest for denne logikken

Reply:
Ja, ser at vi har en teste for dette i isdialogmote som også burde være med her.